### DISCUSSION TITLE
June 2023 Community Call

### DISCUSSION THREAD

**Discussion by heybeckyv** [ucan-wg/spec#155] [Community Call]
Sign up: https://lu.ma/ucan

### Suggested Agenda

- 0.10 release
  - What's next
  - 1.0-RC
  - Focus on quality of live
- rs-ucan updates
- ts-ucan updates
- Alan: DWN
  - https://github.com/decentralized-identity/decentralized-web-node/issues/138
  - https://github.com/decentralized-identity/decentralized-web-node/issues/142
- Sergey: caps DSL

### Video

https://youtu.be/z4u_NPzlBEQ

### Follow-Ups

**Comment by expede** [comment]
- @expede todo: Explicitly no * in invocations
- @cdata suggested a "maturity" table (specs, libs, interop, etc)
- Kind of a bummer that IPLD spec isn't fully up to date
- Alan
  - Be clear about POLA in invocation
  - Revocation: How to handle (eventual) SNARKs?
- Sergey:
  - Capability DSL https://www.craft.me/s/LjrmXahV9qTN0x
